cloudwatch_logs_common/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ endif()
 # Enable strict compiler flags if possible.
 include(CheckCXXCompilerFlag)
 # Removed -Wmissing-declarations until gmock is ignored
-set(FLAGS -pedantic -Wno-long-long -Wall -Wextra -Wcast-align -Wcast-qual -Wformat -Wwrite-strings)
+set(FLAGS -pedantic -Wno-long-long -Wall -Wextra -Wcast-qual -Wformat -Wwrite-strings -Werror -Wcast-align -Wno-error=cast-align)
```

Comment:
I see `-Werror` is already being set at [line 26](https://github.com/aws-robotics/cloudwatch-common/pull/41/files#diff-1340974f2c1a331de5a7bd69f7eae39eR26). Let's not set it here again then.

Reply:
My bad, nice catch! Fixed in the latest push.